### ISSUE TITLE
Defensive file checking stopped working

### ISSUE THREAD

**Issue by donkirkby** [cfe-lab/Kive#761]
Today, we saw NFS problems like we used to have many months ago. We saw generic file reading errors, "Text file busy" errors, and others. The errors seem to happen when one of the three slurm jobs for a run happens on the head node, and another happens on a compute node. Probably, a write operation is still running, when the next step starts.
Maybe I disabled some of the defensive file checking when we moved all the configuration changes into environment variables.
The good news is that each container run happens on a single node, so we shouldn't have this problem once we finish the conversion to container runs.
Get the defensive file checking working again.

**Comment by donkirkby** [comment]
I can't find any places where we removed the defensive file checking. All of the defensive file checking is in the `file_access_utils.py` file in methods with `confirm` in their names. I looked at all of the differences between version 0.12 and 0.13.

    git diff tags/v0.12.2 tags/v0.13.5

Then I searched for the word "confirm" by typing `/confirm`. I didn't find any places where we removed a call.
